sirepo/template/warppba.py

```diff
@@ -307,14 +328,28 @@ def _iteration_title(opmd, data_file):
 
 
 def _opmd_time_series(data_file):
-    prev = None
-    try:
-        prev = main.list_h5_files
-        main.list_h5_files = lambda x: ([data_file.filename], [data_file.iteration])
-        return OpenPMDTimeSeries(py.path.local(data_file.filename).dirname)
-    finally:
-        if prev:
-            main.list_h5_files = prev
+    if sirepo.feature_config.cfg().is_fedora_36:
+        from openpmd_viewer.openpmd_timeseries.data_reader import h5py_reader
+
+        p = h5py_reader.list_files
```

Comment:
I think a comment here explaining what's going on and why and that the code is no longer multi-threaded.

Reply:
I'm not sure I understand what you want. What part of this code was multi-threaded? Also, do you want me to explain the differences between f36 and not or do you want me to just explain what the code inside of the if is doing?

Reply:
Line 338 modifies global state so now the code is not multithreadable. The question I have is "why is it necessary to modify list_files?" Explain that, because it's not obvious.

Reply:
d2ca0f996